tensorflow/python/keras/optimizer_v2/adagrad.py

```diff
@@ -141,7 +141,7 @@ def from_config(cls, config, custom_objects=None):
         An optimizer instance.
     """
     if 'initial_accumulator_value' not in config:
-      config['initial_accumulator_value'] = 0.
+      config['initial_accumulator_value'] = 0.1
```

Comment:
Were was the 0.1 default set for other types?

Reply:
yes, searched for the codebase and all other places use 0.1.